### DISCUSSION TITLE
The term 'clist' is not recognized as the name of a cmdlet, function

### DISCUSSION THREAD

**Discussion by Dev-Negandhi** [lptstr/winfetch#194] [Q&A]
I have been encountering this problem lately. I get the following error code when using winfetch

`& : The term 'clist' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path
is correct and try again.
At C:\tools\winfetch\winfetch.ps1:875 char:24
$chocopkg = (& clist -l)[-1].Split(' ')[0] - 1`

The same issue was also raised in this discussion [here](https://github.com/lptstr/winfetch/discussions/193).

Can anyone guide us why this seems to happen?

**Comment by rashil2000** [comment]
See #189
